app/views/users/show.html.erb

```diff
@@ -1,161 +1,237 @@
 <% content_for :title, "User #{@user.username}" %>
 
 <% if @user.community_user.suspended? %>
-<div class="notice is-danger">
-<% if @user.community_user.suspension_public_comment.nil? %>
-  <p>This user has been <strong>temporarily suspended</strong>.
-<% else %>
-  <p>This user has been <strong>temporarily suspended</strong> <%= @user.community_user.suspension_public_comment %>.
+  <div class="notice is-danger h-m-b-4">
+  <% if @user.community_user.suspension_public_comment.nil? %>
+    <p>This user has been <strong>temporarily suspended</strong>.
+  <% else %>
+    <p>This user has been <strong>temporarily suspended</strong> <%= @user.community_user.suspension_public_comment %>.
+  <% end %>
+
+    The suspension ends <span title="<%= @user.community_user.suspension_end.iso8601 %>">in <%= time_ago_in_words(@user.community_user.suspension_end) %></span>.</p>
+  </div>
 <% end %>
 
-  The suspension ends <span title="<%= @user.community_user.suspension_end.iso8601 %>">in <%= time_ago_in_words(@user.community_user.suspension_end) %></span>.</p>
+<div class="tabs">
+  <%= link_to user_path(@user), class: "tabs--item is-active" do %>
+      <i class="fas fa-user"></i> Profile
+    <% end %>
+  <%= link_to user_activity_path(@user), class: "tabs--item" do %>
+    Activity
+  <% end %>
+  <% if @user == current_user %>
+    <%= link_to edit_user_registration_path, class: "tabs--item" do %>
+      Account
+    <% end %>
+    <%= link_to edit_user_profile_path, class: "tabs--item" do %>
+      Edit
+    <% end %>
+    <%= link_to subscriptions_path, class: "tabs--item" do %>
+      Subscriptions
+    <% end %>
+  <% end %>
 </div>
```

Comment:
This... seems to have been changed back from `render 'tabs'` to having the links hardcoded in the file? You might need to merge develop into your branch, or if not - let's change this back to `render 'tabs'`.

Reply:
Yes. When I’m online (WLAN) again I need to pull and merge.